sdk/src/wallet/utxo.ts

```diff
@@ -113,7 +122,9 @@ export async function createBitcoinPsbt(
                 addressInfo.type,
                 publicKey
             );
-            possibleInputs.push(input);
+            // to support taproot addresses we want to exclude outputs which contain inscriptions
+            if (!inscriptionsSet.has(InscriptionId.toString({ txid: utxo.txid, index: utxo.vout })))
+                possibleInputs.push(input);
```

Comment:
Are you sure that this works? I think the inscriptions [refer to the txids in which they were revealed](https://docs.ordinals.com/inscriptions.html#inscription-ids), rather than the txid they currently reside in:

> The part in front of the i is the transaction ID (txid) of the reveal transaction.

If that's the case then this logic doesn't work.

I think we need to use [`/outputs/<ADDRESS>`](https://docs.ordinals.com/guides/api.html) which has an option to "return UTXOs not containing inscriptions or runes"

Reply:
It's available [upstream](https://github.com/ordinals/ord/blob/master/src/subcommand/server.rs#L224). Why are we using a fork?

Reply:
> Are you sure that this works? I think the inscriptions [refer to the txids in which they were revealed](https://docs.ordinals.com/inscriptions.html#inscription-ids), rather than the txid they currently reside in:
> 
> > The part in front of the i is the transaction ID (txid) of the reveal transaction.
> 
> If that's the case then this logic doesn't work.
> 
> I think we need to use [`/outputs/<ADDRESS>`](https://docs.ordinals.com/guides/api.html) which has an option to "return UTXOs not containing inscriptions or runes"

updated